.circleci/config.yml

```diff
@@ -9,7 +9,32 @@ checkout:
         git checkout -qf FETCH_HEAD
       fi
 
+executors:
+  py26:
+    docker:
+        # staying on old debian image, because buster dropped libssl1.0-dev, which
+        # is required to compile py2.6 via pyenv
+      - image: circleci/python:2.7.14
+      - image: rabbitmq:3.7.4
+  py27:
+    docker:
+      - image: circleci/python:2.7.14
+      - image: rabbitmq:3.7.4
+  py36:
+    docker:
+      - image: circleci/python:3.6
+      - image: rabbitmq:3.7.4
+
 commands:
+  install_cron:
+    steps:
+      - run:
+          # we install cron to allow use of crontab in tests, but not run it,
+          # because we do mock cron in tests
+          name: Install and run cron
+          command: |
+                   sudo apt-get update
+                   sudo apt-get install cron
```

Comment:
Probably best to do `sudo apt-get install -y cron` in case they add a dependency on cron later, which will then make this hang.